source/includes/_feature-importing.md

```diff
@@ -31,7 +31,21 @@ ds <- newDatasetFromFile("my.csv", name="my survey")
 # All three steps are handled within newDatasetFromFile
 ```
 
-POST a Dataset Entity to the datasets catalog.
+POST a Dataset Entity to the datasets catalog. The available initial attributes
+to create a new dataset entity are (all optional):
+
+
+Name | Type | Description
+---- | ---- | -----------
+name | string | Human-friendly string identifier.
```

Comment:
Yes, it will use a default name: "New Dataset". This happens on the tests all the time, we just don't really use/see it.